openqabot/types/aggregate.py

```diff
@@ -84,16 +84,20 @@ def __call__(
             test_repos = defaultdict(list)
 
             # Temporary workaround for applying the correct architecture on jobs, which use a helper VM
+            issues_arch = arch
```

Comment:
maybe better:

`issues_arch' = self.settings.get("TEST_ISSUES_ARCH", arch)`

Reply:
Amended, VR is in progress.